### ISSUE TITLE
Can I use this to get Authorize.Net payment processor in LatePoint?

### ISSUE THREAD

**Issue by tyhallcsu** [latepoint/latepoint-addon-starter#1]
Please let me know if this project is a good place to start to enable Authorize.net payment processing in LatePoint -- thanks!

**Comment by usaphp** [comment]
Yes, this is made specifically for integrating custom modules for LatePoint
